src/components/Services.jsx

```diff
@@ -85,7 +90,11 @@ Services.propTypes = {
 }
 
 const query = client =>
-  client.find('io.cozy.apps.suggestions').where({ silenced: false })
+  client
+    .find('io.cozy.apps.suggestions')
+    .where({ silenced: false })
+    .sortBy([{ slug: 'asc' }])
+    .indexFields(['slug', 'silenced'])
```

Comment:
indexFields order matters. Since `silenced` can decrease the number of suggestions, we should start by this field, no?